lib/internal/readline/interface.js

```diff
@@ -415,6 +416,23 @@ class Interface extends InterfaceConstructor {
     });
   }
 
+  /**
+   * Sets the multiline prompt.
+   * @param {string} prompt
+   * @returns {void}
+   */
+  setMultilinePrompt(prompt) {
+    this[kMultilinePrompt].description = prompt;
+  }
+
+  /**
+   * Returns the current multiline prompt.
+   * @returns {string}
+   */
+  getMultilinePrompt() {
```

Comment:
This is adding these methods to the readline interface, while we currently only use this in the REPL. I don't think we should regularly expose this API.

Since we already have the symbol anyway, I think we can just use those and directly set the symbol and remove this getter and setter.